### ISSUE TITLE
Using drivers

### ISSUE THREAD

**Issue by shizzr** [nomelif/Audionodes#25]
Hey there!
I'm new to GitHub, so I'm not sure if this is the right place for questions/feedback like this.

First of all; great addon! I just downloaded it and played around with it a bit. As far as I can tell, it has huge potential! : )

Is it possible to use blenders drivers to change parameters in Audionodes? I would love to synchronize visual motion with generated audio, but I have not found a way to do so yet.

**Comment by ollpu** [comment]
Feedback and feature requests are definitely welcome here.
I responded to your YouTube comment already but I'll reiterate here: drivers and keyframes don't currently work, and getting them to work on custom properties that are not linked to objects doesn't seem so easy. Some kind of hacky workaround would be needed.

**Comment by shizzr** [comment]
Oh by the way, I found a guy on Youtube that created something interesting. He alters different parameters of the audio nodes by moving a mesh, for example. 
https://youtu.be/yQXKZs10uKY?t=720

It's definitely not the easiest way, but I thought it might interest you : )

**Comment by ollpu** [comment]
Yep, that would be @enzyme64. A kind of output support into Audionodes was added to Sverchok a while back, but I would count that as a "hacky workaround" as they're not actually drivers. Fun to play with nevertheless.

**Comment by enzyme69** [comment]
I made an update for 2020 experiment using Audio Nodes and SVerchok:
https://youtu.be/Qa9A3Y6wtH0

*enzyme69

I am still trying to figure out how to get clean audio from internal Audio Nodes... so far soundflower and blackhole gives me nothing... :/
